Makefile

```diff
@@ -81,9 +81,6 @@ install-develop: clean-build clean-pyc ## install the package in editable mode a
 
 .PHONY: lint-deepecho
 lint-deepecho: ## check style with flake8 and isort
```

Comment:
do we need this make command if it doesn't do anything?

Reply:
Removed this and another unused lint command:
https://github.com/sdv-dev/DeepEcho/pull/94/commits/e4957e79af60a2902c2fce59d9035cc11af933b4